Tests/DependencyInjection/BazingaHateoasExtensionTest.php

```diff
@@ -50,6 +51,16 @@ public function testLoad()
         );
     }
 
+    public function testRelationProviderPass()
+    {
+        $container = $this->getContainerForConfig(array(array()));
+        $container->addCompilerPass(new RelationProviderPass());
```

Comment:
Would have preferred to put this in `Tests/DependencyInjection/Compiler/RelationProviderTest` but would then need to create a new abstract `TestCase` class with the clean up methods. Can do this if required.